pkg/common/consumer/consumer_manager.go

```diff
@@ -345,14 +290,14 @@ func (m *kafkaConsumerGroupManagerImpl) startConsumerGroup(lock *commands.Comman
 }
 
 // getGroup returns a group from the groups map using the groupLock mutex
-func (m *kafkaConsumerGroupManagerImpl) getGroup(groupId string) *managedGroup {
+func (m *kafkaConsumerGroupManagerImpl) getGroup(groupId string) managedGroup {
 	m.groupLock.RLock()
 	defer m.groupLock.RUnlock()
 	return m.groups[groupId]
 }
 
 // getGroup associates a group with a groupId in the groups map using the groupLock mutex
-func (m *kafkaConsumerGroupManagerImpl) setGroup(groupId string, group *managedGroup) {
+func (m *kafkaConsumerGroupManagerImpl) setGroup(groupId string, group managedGroup) {
```

Comment:
comment is "getGroup" for function "setGroup"

Reply:
I can never win, heh.